### PR TITLE
Export GND directive

### PR DESCRIPTION
**Pull request by tobiasschweizer**
resolves https://dasch.myjetbrains.com/youtrack/issue/DSP-579

### CODE REVIEW

File: src/app/action-playground/action-playground.component.html [hunk 1]
```diff
@@ -87,3 +87,6 @@
     <p style="display: inline-block; padding-left: 2%"> {{confirmationDialogResponse}} </p>
 </div>
 
+<p>GND Directive (Integrated Authority File)</p>
```
Comment (mdelez) [inline src/app/action-playground/action-playground.component.html:90]: Please add an `<hr>` here :)

**Review by mdelez — CHANGES_REQUESTED** [review]
Works for me, just a small change requested

File: src/app/action-playground/action-playground.component.html (hunk 1 above)
Comment (tobiasschweizer) [reply src/app/action-playground/action-playground.component.html:92]: done in https://github.com/dasch-swiss/dsp-ui-lib/pull/162/commits/1a9266c3a439a124f73b689c83f30443ede2b462

**Review by mdelez — APPROVED** [review]
Thanks!

**Pull request merged**